lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
Unfortunately, the Node.js app has no tty, so there is no way to implicitly determine if STDIN should be used. So we either fully use STDIN, so launching a default Owens Desktop would look like:

```sh
echo "{}" | bin/rake batch_connect:new_session RAILS_RELATIVE_URL_ROOT=dev/ood-dashboard BC_APP_TOKEN=sys/bc_desktop_v2/owens
```

or we provide an explicit way to turn on or off STDIN checking.